man/sg.1.xml

```diff
@@ -70,6 +75,38 @@
     </para>
   </refsect1>
 
+  <refsect1 id='options'>
+    <title>OPTIONS</title>
+    <para>
+      The options which apply to the <command>sg</command> command are:
+    </para>
+    <variablelist remap='IP'>
+      <varlistentry>
+	<term><option>-</option>, <option>-l</option></term>
+	<listitem>
+	  <para>
+	    Provide an environment similar to what the user would expect had
+	    the user logged in directly.
```

Comment:
Please use semantic newlines.  See man-pages(7).

In this case, I'd break the lines as:

```
Provide an environment
similar to what the user would expect
had the user logged in directly.
```

Reply:
Adjusted